x-pack/plugins/lens/public/editor_frame_service/embeddable/embeddable_component.tsx

```diff
@@ -43,11 +51,39 @@ export type TypedLensByValueInput = Omit<LensByValueInput, 'attributes'> & {
     | LensAttributes<'lnsMetric', MetricState>;
 };
 
-export type EmbeddableComponentProps = TypedLensByValueInput | LensByReferenceInput;
+export type EmbeddableComponentProps = (TypedLensByValueInput | LensByReferenceInput) & {
+  withActions?: boolean;
+};
+
+interface PluginsStartDependencies {
+  uiActions: UiActionsStart;
+  embeddable: EmbeddableStart;
+  inspector: InspectorStartContract;
+}
 
-export function getEmbeddableComponent(embeddableStart: EmbeddableStart) {
+export function getEmbeddableComponent(core: CoreStart, plugins: PluginsStartDependencies) {
   return (props: EmbeddableComponentProps) => {
+    const { embeddable: embeddableStart, uiActions, inspector } = plugins;
     const factory = embeddableStart.getEmbeddableFactory('lens')!;
-    return <EmbeddableRenderer factory={factory} input={props} />;
+    // by default hide the time badges. Consumers can always override via disabledActions prop
+    const input = { disabledActions: ['CUSTOM_TIME_RANGE_BADGE'], ...props };
```

Comment:
Why do we want to hide it by default? Is it a breaking change?

```suggestion
    const input = { disabledActions: ['CUSTOM_TIME_RANGE_BADGE'], ...props };
```

Reply:
Yes, probably this way it sounds like a breaking change. Maybe I could prevent the badge/notifications rendering via a special prop on the `EmbeddablePanel` itself